Public/Src/Engine/Scheduler/Tracing/ExecutionLog.Events.cs

```diff
@@ -1205,6 +1205,12 @@ public struct StatusEventData : IExecutionLogEventData<StatusEventData>
         /// </summary>
         public int UnresponsivenessFactor;
 
+        /// <nodoc />
+        public long ProcessPipsPending;
+
+        /// <nodoc />
+        public long ProcessPipsRunning;

```

Comment:
Please doc exactly what these mean. Does Running mean the running state? an external child process actually executing? etc